software/webdb.yml

```
@@ -0,0 +1,39 @@
+---
+name: Addition
+about: Add new software to the list.
+title: Add WebDB
+labels: addition
+assignees: ''
+
+---
+
+Thanks for taking the time to suggest an addition to awesome-selfhosted! 
+
+Please fill out information below (all fields are mandatory unless noted otherwise):
+
+```yaml
+name: "WebDB"
+website_url: "https://webdb.app"
+source_code_url: "https://gitlab.com/web-db/app"
+description: "Open Source and Efficient Database IDE"
+licenses:
+  - AGPL-3.0
+platforms:
+  - Docker
+tags:
+  - Database
+  - IDE
+  - AI
+  - Versionning
+depends_3rdparty: false
+demo_url: "https://demo.webdb.app/"
+```
+
+To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):
+- [x] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
+- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
+- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
+- [x] Any software project you are adding to the list is actively maintained.
+- [x] Any software project you are adding was first released more than 4 months ago.
+- [x] Any software project you are adding has working installation instructions.
+- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
```

```suggestion
To ensure your issue is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull issue. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
```
These are not necessary as you already did it in the comment.